Packs/Edgescan/Integrations/Edgescan/README.md

```diff
@@ -0,0 +1,5110 @@
+Cloud-based continuous vulnerability management and penetration testing solution.
+This integration was integrated and tested with version xx of Edgescan
```

Comment:
Is there a version of Edgescan? If not, please remove this line

Reply:
added v1.6